### DISCUSSION TITLE
Where are the CAM recordings stored?

### DISCUSSION THREAD

**Discussion by rgroothuis** [marcone/teslausb#781] [Q&A] [answered]
I've successfully configures and installed the TeslaUSB image on a Raspberry zero. I'm now wondering where the CAM recordings are stored on the Raspberry. Any suggestions what the location is on the Raspberry zero? Which mount path?

**Comment by marcone** [comment, marked as answer]
They are stored in a disk image (/backingfiles/cam_disk.bin) which is either exposed to the car via the USB mass storage driver, or mounted on /mnt/cam when the Pi needs to access it. I recommend not doing both at the same time.
